po/ru.po

```diff
@@ -1780,13 +1950,60 @@ msgid ""
 "   run `cargo` commands, it will automatically download and compile missing\n"
 "   dependencies for you."
 msgstr ""
+"1. Нажмите кнопку ”Copy to clipboard” на примере кода, который вы хотите скопировать.\n"
+"\n"
+"2. Используйте`cargo new exercise`, чтобы создать новую директорию `exercise/` для вашего кода:\n"
+"\n"
+"    ```shell\n"
+"    $ cargo new exercise\n"
+"         Created binary (application) `exercise` package\n"
+"    ```\n"
+"\n"
+"3.  Перейдите в директорию `exercise/` и используйте `cargo run` для сборки и запуска вашего двоичного файла:\n"
+"\n"
+"    ```shell\n"
+"    $ cd exercise\n"
+"    $ cargo run\n"
+"       Compiling exercise v0.1.0 (/home/mgeisler/tmp/exercise)\n"
+"        Finished dev [unoptimized + debuginfo] target(s) in 0.75s\n"
+"         Running `target/debug/exercise`\n"
+"    Hello, world!\n"
+"    ```\n"
+"\n"
+"4. Замените шаблонный код в `src/main.rs` своим кодом. Например, используя пример кода с предыдущей страницы,\n"
+"сделайте `src/main.rs` похожим на\n"
+"\n"
+"    ```rust\n"
+"    fn main() {\n"
+"        println!(\"Edit me!\");\n"
+"    }\n"
+"    ```\n"
+"\n"
+"5. Используйте `cargo run` для сборки и запуска вашего обновленного двоичного файла:\n"
+"\n"
+"    ```shell\n"
+"    $ cargo run\n"
+"       Compiling exercise v0.1.0 (/home/mgeisler/tmp/exercise)\n"
+"        Finished dev [unoptimized + debuginfo] target(s) in 0.24s\n"
+"         Running `target/debug/exercise`\n"
+"    Edit me!\n"
+"    ```\n"
+"\n"
+"6. Используйте `cargo check` для быстрой проверки вашего проекта на наличие ошибок, используйте `cargo build` для\n"
+"компиляции проекта не запуская его. Вы найдете результат в директории `target/debug/` для отладочной сборки.\n"
+"Используйте `cargo build --release` для создания оптимизированной релизной сборки в `target/release/`.\n"
+"\n"
+"7. Вы можете добавить зависимости для вашего проекта, отредактировав `Cargo.toml`.\n"
```

Comment:
Hi @ramosian-glider, let me introduce you to GitHub suggestions:

```suggestion
"7. Вы можете добавить зависимости для вашего проекта, отредактировав файл `Cargo.toml`.\n"
```

I discovered this feature recently myself 🙂 It saves dramatically on the round trip times since the PR author can apply the suggestions with a single click.

Reply:
Great, thanks for pointing this out!
The text may still need to be reflown after applying the suggestions (are there line length limits here?)